### PR TITLE
checkRouteTransit checks if the route is transit

### PR DESCRIPTION
**Pull request by petermarathas**
If there's no bus to get you somewhere, google tells you to walk. This instead shows a message saying there's no bus.

### CODE REVIEW

File: www/pages/plan-trip/plan-trip-controller.js [hunk 1]
```diff
@@ -320,6 +320,10 @@ angular.module('pvta.controllers').controller('PlanTripController', function ($s
     });
   };
 
+  var checkRouteTransit = function(route) {
```
Comment (akaplo) [inline www/pages/plan-trip/plan-trip-controller.js]: You can/should define as `function checkRouteTransit (route)` for consistency with the rest of the app.

Comment (akaplo) [reply www/pages/plan-trip/plan-trip-controller.js]: Could it be renamed?  If I didn't know what it did, I wouldn't know what "check route transit" means.

**Review by akaplo — CHANGES_REQUESTED** [review]
My 2 comments and CC failures ("similar code" failures are OK for now).  Simple and nice!

**Comment by petermarathas** [comment]
@akaplo <code>checkRouteTransit</code> changed to <code>function confirmValidRoute</code>.

**Comment by akaplo** [comment]
Schweet. I'll approve, but fix your two CC issues before you merge please!

**Review by akaplo — APPROVED** [review]

**Pull request merged**